Makefile

```diff
@@ -249,7 +250,7 @@ man/%: docs/man/%.md FORCE
 	go-md2man -in "$<" -out "$@"
 
 define installmanpage
-mkdir -p $(DESTDIR)/man/man$(2);
+$(INSTALL) -d $(DESTDIR)/man/man$(2);
```

Comment:
Note to other curious reviewers: install's -d flag is not GNU-only. It is supported by [FreeBSD as well](https://www.freebsd.org/cgi/man.cgi?query=install).